whois-client/src/main/java/net/ripe/db/whois/query/QueryMessages.java

```diff
@@ -62,6 +60,13 @@ public static Message unvalidatedAbuseCShown(final CharSequence key, final CharS
                 "\nAbuse-mailbox validation failed. Please refer to %s for further information.", key, value, orgId);
     }
 
+    public static Message roaRouteConflicts(final long asn){
+        return new QueryMessage(Type.INFO, ""
```

Comment:
We are thinking of severirty in messages than rather have it as warning?

Reply:
I don't understand what you mean

Reply:
Done, as discussed